Fix default model path for it to be found outside the source directory
Author: Cintia Willemyns (Vrije Universiteit Brussel)
--- a/predict.py	2023-11-15 14:41:11.760297000 +0100
+++ b/predict.py	2023-11-15 15:00:42.079686000 +0100
@@ -13,7 +13,7 @@
 parser.add_argument('--no-pdb', dest="no_pdb", action="store_true")
 parser.add_argument('--no-distance-map', dest="no_distance_map", action="store_true")
 parser.add_argument('--save-distance-array', dest="save_distance_array", action="store_true")
-parser.add_argument('-m', '--model', default="model/EMBER3D.model", dest='model_checkpoint', type=str)
+parser.add_argument('-m', '--model', default=os.path.join(os.path.dirname(__file__), 'model', 'EMBER3D.model') , dest='model_checkpoint', type=str)
 parser.add_argument('--t5-dir', dest='t5_dir', default="./ProtT5-XL-U50/", type=str)
 args = parser.parse_args()
 
